### ISSUE TITLE
Albert's Task 1/06/2024 - 1/23/2024

### ISSUE THREAD

**Issue by natt96z** [natt96z/cybersac#141]
*Work on this week's milestones under the issues feature on GitHub (Milestone Admin)
*Travel to Required Schools For CyberPatriots lectures (RSVP: MSA/Valley High Schools)
*Social Media Posting - Get Useful Promotional Material  (Photos of work, tech tips etc)
*Work On CyberTraining Material Below for 12/11/2023 (Valley High)
*Ask MSA if they are willing to re-partitipate in program. (@userliluzibert , Nate)
*Site Feedback Based on Student Interaction 

**IMPORTANT DATES - END OF MONTH CENTURY HIGH SCHOOL  @ SANTA ANA COLLEGE**
Event dates: January 16th, 19th, and 29th  
ROOM: 215
Student count: 70 students each day  
Presentation slot1: 11:05am to 11:35am  
Presentation slot2: 11:45am to 12:15pm

**_Note from Nate: Professor Vu expects us back second week of January, beofre that happens we need to meet on Zoom this thursday @3PM. All needed reveiw material has also bee added top the new CyberTraining section, please review this matrerial.

OH and **CyberPassXVI** is VM login
----------------------------------------------------------------------------------------------------------------------
**_*[CyberTraining] Cisco Packet Tracer Part 3 Lecture Tutorial 3_**

**Link:** https://cybersac.readthedocs.io/en/latest/HardExpertCyberSAC.html#cisco-packet-tracer-cyberlab

**_*[CyberTraining] Discuss useful basic Ubuntu Linux Commands_**

**Link:** https://cybersac.readthedocs.io/en/latest/MediumCybersac.html#useful-basic-linux-commands

**_*[CyberTraining] Work On Ubuntu VM Scripting_** 

**Link:** https://cybersac.readthedocs.io/en/latest/HardExpertCyberSAC.html#hardened-debian-gnu-linux-distro-auditing

**_*Review Windows 10 CyberPatriot Lecture Again (If time allows)_**
https://cybersac.readthedocs.io/en/latest/MediumCybersac.html

**Comment by liluzibird** [comment]
>  Student count: 70 students each day

I think you mean 7

**Comment by natt96z** [comment]
> > Student count: 70 students each day
> 
> I think you mean 7

Oh its 70, divided it seems.

**Comment by natt96z** [comment]
![image](https://github.com/natt96z/cybersac/assets/112590005/dc6adf4c-8116-4b93-89c0-297b30175c7e)
![560798-middle](https://github.com/natt96z/cybersac/assets/112590005/f7b9eacb-178a-46eb-b85e-80fab5c237bf)


_**1/8/2024: New Milestones Added - Lecture Section (Faster Set-Up Procedure)**_
Team together agreed that presentation set-up takes too long to accomplish. Now lesson plans needed for that given week are included in the Milestones/Issues section as needed. This also includes calendar days and other useful instructional material that will also drive questions and discussion on GitHub and FluffyChat.

**Comment by natt96z** [comment]
![image](https://github.com/natt96z/cybersac/assets/112590005/5026dafb-846c-4cf5-90c4-6c85dd4fca41)
![image](https://github.com/natt96z/cybersac/assets/112590005/090e11cd-a454-421e-b7e0-a0d846ed4090)

**_1/9/2024: Updated Powerpoint Material - Minor Site Maintenance_**
Me and @liluzibird noticed power-points failed to load correctly and some were outdated as well, so we decided to re-add them to the documentation website and do some minor code cleanup as well. This is well needed since we can easily point the students to the material during our training meetings or when they study on there own. Last lecture some failed to load, that issue has now been fixed.

**Comment by natt96z** [comment]
![image](https://github.com/natt96z/cybersac/assets/112590005/cd7cf7d1-efdd-41de-b958-a5e613a4813a)

![image](https://github.com/natt96z/cybersac/assets/112590005/3ba5b431-d53f-4d7b-8a3f-44e8be172e2b)

**_1/10/2023 Zoom Meeting W/ Alicia Regarding CyberPatriots Material and Next Weeks Event Schedule_**

On this day while not exactly on schedule with the schools we are coaching we got together in a quick hour long meeting to discus the room layout, lecture material and other processes for next weeks Cyber event. We learned the class load of 70 students and how they would be split up at the SAC campus. @liluzibird @computerbluerain @natt96z

**Comment by natt96z** [comment]
![image](https://github.com/natt96z/cybersac/assets/112590005/53f1fdfb-c3e1-4b00-92b0-0b88107365e7)


1/11/2023 Jitsi Meeting With Nate Over Simplifying Virtual Machines For Next Week
So as requested we are going to come early 8AM at SAC Tuesday next week with new formatted virtual machines, more work will be done next Monday as well to make them easy to use.

**Comment by natt96z** [comment]
**1/18/2024 Work at Valley High School W/Students**
We did a lot of great work today, especially when dealing with Windows OS configuration with security policies and more advanced Cisco packet tracer lessons as well, as very network-oriented lesson plans we made sure they also understand Linux VM as well.

![image](https://github.com/natt96z/cybersac/assets/112590005/fa7bbfd1-34e5-4b70-a22a-61a4c1a7311e)

**1/19/2024 - 1/22/2024 - Completed At School SAC Cyber Event**
This is our second CyberPatriots event showing the students the Cisco Packet tracer software led by @computerbluerain. This event focused on the importance of network configuration and, more importantly, security. We made sure to come in 1-2 hours early again to prepare to set up the desktop environment in classroom 215.


![image](https://github.com/natt96z/cybersac/assets/57073805/0089e013-d3ae-4db2-86a6-53c149c66ede)

edited picture of students
